playbooks/katello_devel.yml

```diff
@@ -13,7 +13,4 @@
       - "--disable-system-checks"
       - "--katello-devel-enable-ostree=true"
   roles:
-    - etc_hosts
-    - foreman_server_repositories
-    - foreman_installer_devel_scenario
-    - foreman_installer
+    - katello_devel
```

Comment:
Sorry just occurred to me, the `vars` section in this playbook. Is that needed anymore with the meta role?